src/commands/apply/index.ts

```diff
@@ -125,7 +126,7 @@ export default class Apply extends Command {
       spaceId: flags.space,
       environmentId: flags.environment,
       inputPath: flags.file,
-      changeset: createChangeset(flags.source, flags.target),
```

Comment:
Curious why this was being done like this before 🤔

Reply:
Maybe to make Typescript happy - we used to parse the changeset only after this point in the code until recently